src/web/src/components/SearchResults.tsx

```diff
@@ -85,7 +90,7 @@ const SearchResults = () => {
     !isReachingEnd && data && size > 0 && typeof data[size - 1] === 'undefined';
 
   // If there is no posts or if the search bar is empty, then show the search help, otherwise hide it
-  if (!error && (isEmpty || textParam.length === 0)) {
+  if (!error && (isEmpty || (post.length === 0 && author.length === 0))) {
```

Comment:
This is causing the search page to immediately search upon a letter being entered which is what we don't want

Reply:
I thought it was a feature :)